### PR TITLE
Move examples from WaterLily/examples

### PR DESCRIPTION
**Pull request by marinlauber**
I moved the examples from WaterLily/examples here and combined it with [#136](https://github.com/WaterLily-jl/WaterLily.jl/pull/136).

I have added the pressure solver logger here as well. I am not sure how we want to do this. Maybe the plot utils should go into an extension of Waterlily, like `WaterLilyPlotsExt.jl` and `WaterLilyMakieExt.jl`, and the examples then only need to import those to have access to `sim_gif` and so on.

### CODE REVIEW

**Comment by marinlauber** [comment]
@weymouth @b-fg how do we feel about this one?

**Comment by b-fg** [comment]
This is a good idea. Some things:
- Use a consistent repo name wrt to other repos in the organisation, for example `WaterLily.jl-Examples`. Note that (in my opinion) these are not really tutorials, those would be Pluto notebooks for example.
- Update README and examples according to https://github.com/WaterLily-jl/WaterLily.jl/pull/148, since now this is a bit behind master.

**Comment by marinlauber** [comment]
@b-fg Just changed the `README.md` and updated the examples as well.

File: Manifest.toml
Comment (b-fg) [inline Manifest.toml:1]: This should not be included. See the WaterLily.jl .gitignore.

Comment (marinlauber) [reply Manifest.toml:1]: It is in the `.gitignore`, I just forgot to `git rm` the file...

**Comment by weymouth** [comment]
There are two pluto notebooks. And we could link to my youtube tutorial
videos in the README. @marin, you have been teaching this and working with
other users. What do you think would be the most useful for others?

Gabriel D Weymouth
_______________________________________________
"Computers are useless. They can only give you answers." Pablo Picasso

**Comment by weymouth** [comment]
Looks good to me.

**Review by weymouth — COMMENTED** [review]
Happy to move this

**Pull request merged**